DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/ipv4/conftest.py

```diff
@@ -11,32 +11,32 @@
 
 @pytest_asyncio.fixture()
 async def enable_ipv4_forwarding(testbed):
-    tgen_dev, dent_devices = await tgen_utils_get_dent_devices_with_tgen(testbed, [], 4)
-    if not tgen_dev or not dent_devices:
-        print('The testbed does not have enough dent with tgen connections')
-        return
-    dent = dent_devices[0].host_name
+    _, dent_devices = await tgen_utils_get_dent_devices_with_tgen(testbed, [], 0)
```

Comment:
can we use here `tb_get_all_devices`? from **tb_utils**?

Reply:
probably yes, but '*_get_dent_devices_with_tgen' also contains some validation and can filter out devices, that we don't want (dist)